Dockerfile.clang

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:22.04
+WORKDIR /go/src/github.com/cilium/tetragon
+RUN apt-get update
+RUN apt-get -y upgrade
+RUN apt-get install -y --no-install-recommends clang-13 llvm-13 make
```

Comment:
I wonder if it makes sense to install a specific version of clang here to avoid surprises. E.g., `sudo apt install clang-13=1:13.0.1-7`. (We can have it as an ARG in the Dockerfile)

Reply:
I pushed new version with this, please check